CHANGELOG.rst

```diff
@@ -29,6 +29,8 @@ Changes:
 - Added ``OpenSSL.SSL.Connection.get_verified_chain`` to retrieve the
   verified certificate chain of the peer.
   `#894 <https://github.com/pyca/pyopenssl/pull/894>`_.
+- Allow omission of the verification callback in ``Context.set_verify``.
```

Comment:
One last request, please document that this will use openssl’s default verification both here and anywhere else we need.

Reply:
done!